src/types/entities.ts

```diff
@@ -12,8 +12,9 @@ export type VaultTransaction = {
 
 export type FormattedVaultTransaction = Pick<
   VaultTransaction,
-  "id" | "vaultAddress" | "depositerAddress" | "type"
+  "id" | "vaultAddress" | "depositerAddress"
 > & {
+  type: string;
```

Comment:
should this not also be `withdraw | deposit` ?

Reply:
I make the type looser to allow mapping string into formatted value.
Now that you mention it, i prob. better to leave it strict and map it right on the View component